interface/src/app/tribe_auth/resource.js

```diff
@@ -0,0 +1,14 @@
+/*
+ * The resources for fetching the OAuth2 token from the tribe_client url
+ */
+angular.module("adage.tribe_auth.resource", ['ngResource'])
+    .factory('User', ['$resource', function($resource) {
+        return $resource('/tribe_client/return_user', {}, {
```

Comment:
If you want, you can simplify the code into:
```
return $resource('/tribe_client/return_user', {}, {}
)};
```
Then in line #15 of **tribe_user.js**, instead of calling `User.query(...)`, call `User.get(...)`.